pkg/controller/cassandracluster/pod.go

```diff
@@ -133,9 +133,9 @@ func (rcc *ReconcileCassandraCluster) hasUnschedulablePod(namespace string, dcNa
 	for _, pod := range podsList.Items {
 		if pod.Status.Phase != v1.PodRunning && pod.Status.Conditions != nil {
 			for _, cs := range pod.Status.ContainerStatuses {
-				if cs.Ready == false &&
-					cs.State.Waiting != nil && cs.State.Waiting.Reason == "ImagePullBackOff" {
-					//TODO: delete Pod in this case so that it can be scheduled again
+				if cs.Ready == false && cs.State.Waiting != nil && cs.State.Waiting.Reason == "ImagePullBackOff" {
+					//TODO: delete Pod in this case so that it can be scheduled again if image in spec and image in
```

Comment:
Do we have a ticket for that ?

Reply:
I didn't encounter this since long time for now.. not sur a ticket exist on this